lib/Conch/Route/Datacenter.pm

```diff
@@ -180,6 +143,324 @@ __END__
+=item * Request: input.yaml#/DatacenterRoomCreate
+
+=item * Response: Redirect to the created room
+
+=back
+
+=head3 C<GET /room/:datacenter_room_id>
+
+
+=over 4
+
+=item * Requires System Admin Authorization
+
+=item * Response: response.yaml#/DatacenterRoomDetailed
+
+=back
+
+=head3 C<POST /room/:datacenter_room_id>
+
+=over 4
+
+=item * Requires System Admin Authorization
+
+=item * Request: input.yaml#/DatacenterRoomUpdate
+
+=item * Response: Redirect to the updated room
+
+=back
+
+=head3 C<DELETE /room/:datacenter_room_id>
+
+=over 4
+
+=item * Requires System Admin Authorization
+
+=item * Response: C<204 NO CONTENT>
+
+=back
+
+=head3 C<GET /room/:datacenter_room_id/racks>
+
+=over 4
+
+=item * Requires System Admin Authorization
+
+=item * Response: response.yaml#/Racks
+
+=back
+
+=head3 C<GET /rack_role>
+
+=over 4
+
+=item * Requires System Admin Authorization
+
+=item * Response: response.yaml#/RackRoles
+
+=back
+
+=head3 C<POST /rack_role>
+
+=over 4
+
+=item * Requires System Admin Authorization
+
+=item * Request: input.yaml#/RackRoleCreate
+
+=item * Response: Redirect to the created rack role
+
+=back
+
+=head3 C<GET /rack_role/:rack_role_id_or_name>
+
+=over 4
+
+=item * Requires System Admin Authorization
+
+=item * Response: response.yaml#/RackRole
+
+=back
+
+=head3 C<POST /rack_role/:rack_role_id_or_name>
+
+=over 4
+
+=item * Request: input.yaml#/RackRoleUpdate
+
+=item * Response: Redirect to the updated rack role
+
+=back
+
+=head3 C<DELETE /rack_role/:rack_role_id_or_name>
+
+=over 4
+
+=item * Response: C<204 NO CONTENT>
+
+=back
+
+=head3 C<GET /rack>
+
+=over 4
+
+=item * Requires System Admin Authentication
+
+=item * Response: response.yaml#/Racks
+
+=back
+
+=head3 C<POST /rack>
+
+=over 4
+
+=item * Requires System Admin Authentication
+
+=item * Request: input.yaml#/RackCreate
+
+=item * Response: Redirect to the created rack
+
+=back
+
+=head3 C<GET /rack/:rack_id>
+
+=over 4
+
+=item * Response: response.yaml#/Rack
+
+=back
+
+=head3 C<POST /rack/:rack_id>
+
+=over 4
+
+=item * Request: input.yaml#/RackUpdate
+
+=item * Response: Redirect to the updated rack
+
+=back
+
+=head3 C<DELETE /rack/:rack_id>
+
+=over 4
+
+=item * Response: C<204 NO CONTENT>
+
+=back
+
+=head3 C<GET /rack/:rack_id/layouts>
+
+=over 4
+
+=item * Response: response.yaml#/RackLayouts
+
+=back
+
+=head3 C<GET /rack/:rack_id/assignment>
+
+=over 4
+
+=item * Response: response.yaml#/RackAssignments
+
+=back
+
+=head3 C<POST /rack/:rack_id/assignment>
+
+=over 4
+
+=item * Request: input.yaml#/RackAssignmentUpdates
+
+=item * Response: Redirect to the updated rack assignment
+
+=back
+
+=head3 C<DELETE /rack/:rack_id/assignment>
+
+This method requires a request body.
+
+=over 4
+
+=item * Request: input.yaml#/RackAssignmentDeletes
+
+=item * Response: C<204 NO CONTENT>
+
+=back
+
+=head3 C<< POST /rack/:rack_id/phase?rack_only=<0|1> >>
+
+Use the C<rack_only> query parameter to specify whether to only update the rack's phase, or all
```

Comment:
What's the default?

Reply:
Added better text explaining the default is `0`